CONTRIBUTING.md

```diff
@@ -143,6 +156,12 @@ Our build will fail if source code is not formatted according to that style.
 The main goal is to avoid extensive reformatting caused by different IDEs having different opinion
 about how things should be formatted by establishing.
 
+Note for Windows users: if you are seeing all of your files being changed by `./gradlew spotlessApply` (below), it's probably the line endings, and make sure you are using a `core.autocrlf` setting of either `true` or `input`.
+
+```bash
+git config core.autocrlf true
```

Comment:
@trask What do you think about adding `.gitattributes` with something like this?

```
* text=auto eol=lf

*.bat text eol=crlf
*.cmd text eol=crlf
```

I often develop on Windows, and I have only had pain with repositories without gitattributes and have never seen problems after adding it (modern OS / tools handle LF just fine on Windows too, even Notepad!).

Reply:
oh, yes, that does work, as spotless respects the line endings in `.gitattributes`! (https://github.com/diffplug/spotless/issues/23)

Reply:
@heyams let's remove this section and add the `.gitattributes` file suggested by @anuraaga above 👍